src/main/java/com/google/cloud/tools/appengine/operations/AppYamlProjectStaging.java

```diff
@@ -102,6 +105,7 @@ void stageStandardArchive(AppYamlProjectStageConfiguration config)
     copyExtraFiles(config, copyService);
     copyAppEngineContext(config, copyService);
     copyArtifact(config, copyService);
+    copyArtifactJarClasspath(config, copyService);
```

Comment:
what's the effect of this on Java 8?

Reply:
It doesn't get called. This code path is only for app.yaml standard apps.